src/test/java/stringcalculator/StringCalculator.java

```diff
@@ -0,0 +1,90 @@
+package stringcalculator;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+    private static final String OPERATION_REGEX = "[+\\-*/]+";
+    private static final String NUMBER_REGEX = "\\d+";
+    private static final String NOT_VALID_REGEX = "[^\\d+\\-*/^\\s]";
+
+    public static int calculate(String input) {
```

Comment:
```suggestion
    public static int calculate(List<Integer> numbers, List<String> operations) {
```
`calculate` 메소드 안에서 많은 책임을 가지고 있어요. 
숫자도 만들고 문자열도 분석하고, `calculate`가 순수하게 하고자 하는 인자만 주입받도록 고민해보면 좋을듯해요!
(예시는 "인자를 받는다면 어떨까?"에 대한 예시일뿐입니다)